internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/public.go

```diff
@@ -37,6 +37,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+// QuicklinkName is the reserved name for a quicklink.
+// Creating (or updating) a link with (to) the same name will be blocked by the server
+var QuicklinkName = "Quicklink"
```

Comment:
You should make a constant out of that.

Reply:
Good point :+1: will do so

Any feedback on the name itself? `"Internal Quicklink"` could make it less likely for users to accidentally run into the bad request. Or we just use a fixed uuid?
cc @micbar

Reply:
I guess the name needs to be decided by PM.
Also not important for this ticket but in the future the name probable needs to be translatable. :see_no_evil:

Reply:
Is that not the name shown by web?

Reply:
No, in the mockups I saw the quicklink doesn't show a name at all